### ISSUE TITLE
'dragonfire not found' after a successful .deb installation

### ISSUE THREAD

**Issue by Multiboxer** [DragonComputer/Dragonfire#73]
No errors occur during installation via the latest 0.9.9.deb however running Dragonfire outputs:
`/usr/bin/dragonfire: 3: /usr/bin/dragonfire: /usr/share/python/dragonfire/bin/dragonfire: not found
`.

The installer creates no folder named `/usr/share/python/dragonfire/` and the closest I can find is `/usr/lib/python2.7/dist-packages/dragonfire/` I am running Ubuntu 16.04 with python 2.7.14 & 3.6.5 and **my best guess is there is some confusion in regards as to which python version the installer wants to use**. The required dependencies are installed in python 3.6 and not 2.7 however all Dragonfire related files are installed in `/usr/lib/python2.7/dist-packages/` however the installation process mentions python3.6 here:
`/usr/local/lib/python3.6/site-packages/en_core_web_sm -->`
`/usr/local/lib/python3.6/site-packages/spacy/data/en`
and here:
`Requirement already up-to-date: PyAudio in /usr/local/lib/python3.6/site-packages (0.2.11)
Requirement already up-to-date: wikipedia in /usr/local/lib/python3.6/site-packages (1.4.0)
Requirement already up-to-date: PyUserInput in /usr/local/lib/python3.6/site-packages (0.1.11)
Requirement already up-to-date: tinydb in /usr/local/lib/python3.6/site-packages (3.9.0.post1)
Requirement already up-to-date: youtube_dl in /usr/local/lib/python3.6/site-packages (2018.5.18)
Requirement already up-to-date: spacy in /usr/local/lib/python3.6/site-packages (2.0.11)
Requirement already up-to-date: pyowm in /usr/local/lib/python3.6/site-packages (2.8.0)
Requirement already up-to-date: tensorflow==1.0.0 in /usr/local/lib/python3.6/site-packages (1.0.0)
Requirement already up-to-date: deepspeech==0.2.0a5 in /usr/local/lib/python3.6/site-packages (0.2.0a5)
Requirement already up-to-date: tweepy in /usr/local/lib/python3.6/site-packages (3.6.0)
Requirement already up-to-date: metadata_parser in /usr/local/lib/python3.6/site-packages (0.9.19)
Requirement not upgraded as not directly required: beautifulsoup4 in /usr/local/lib/python3.6/site-packages (from wikipedia) (4.6.0)
Requirement not upgraded as not directly required: requests<3.0.0,>=2.0.0 in /usr/local/lib/python3.6/site-packages (from wikipedia) (2.18.4)
Requirement not upgraded as not directly required: python-xlib in /usr/local/lib/python3.6/site-packages (from PyUserInput) (0.23)
Requirement not upgraded as not directly required: numpy>=1.7 in /usr/local/lib/python3.6/site-packages (from spacy) (1.14.3)
Requirement not upgraded as not directly required: murmurhash<0.29,>=0.28 in /usr/local/lib/python3.6/site-packages (from spacy) (0.28.0)
Requirement not upgraded as not directly required: cymem<1.32,>=1.30 in /usr/local/lib/python3.6/site-packages (from spacy) (1.31.2)
Requirement not upgraded as not directly required: preshed<2.0.0,>=1.0.0 in /usr/local/lib/python3.6/site-packages (from spacy) (1.0.0)
Requirement not upgraded as not directly required: thinc<6.11.0,>=6.10.1 in /usr/local/lib/python3.6/site-packages (from spacy) (6.10.2)
Requirement not upgraded as not directly required: plac<1.0.0,>=0.9.6 in /usr/local/lib/python3.6/site-packages (from spacy) (0.9.6)
Requirement not upgraded as not directly required: pathlib in /usr/local/lib/python3.6/site-packages (from spacy) (1.0.1)
Requirement not upgraded as not directly required: ujson>=1.35 in /usr/local/lib/python3.6/site-packages (from spacy) (1.35)
Requirement not upgraded as not directly required: dill<0.3,>=0.2 in /usr/local/lib/python3.6/site-packages (from spacy) (0.2.7.1)
Requirement not upgraded as not directly required: regex==2017.4.5 in /usr/local/lib/python3.6/site-packages (from spacy) (2017.4.5)
Requirement not upgraded as not directly required: six>=1.10.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.0.0) (1.11.0)
Requirement not upgraded as not directly required: protobuf>=3.1.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.0.0) (3.5.2.post1)
Requirement not upgraded as not directly required: wheel>=0.26 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.0.0) (0.31.1)
Requirement not upgraded as not directly required: scipy in /usr/local/lib/python3.6/site-packages (from deepspeech==0.2.0a5) (1.1.0)
Requirement not upgraded as not directly required: PySocks>=1.5.7 in /usr/local/lib/python3.6/site-packages (from tweepy) (1.6.8)
Requirement not upgraded as not directly required: requests-oauthlib>=0.7.0 in /usr/local/lib/python3.6/site-packages (from tweepy) (0.8.0)
Requirement not upgraded as not directly required: urllib3<1.23,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.0.0->wikipedia) (1.22)
Requirement not upgraded as not directly required: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.0.0->wikipedia) (2018.4.16)
Requirement not upgraded as not directly required: idna<2.7,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.0.0->wikipedia) (2.6)
Requirement not upgraded as not directly required: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.0.0->wikipedia) (3.0.4)
Requirement not upgraded as not directly required: wrapt in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (1.10.11)
Requirement not upgraded as not directly required: tqdm<5.0.0,>=4.10.0 in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (4.23.3)
Requirement not upgraded as not directly required: cytoolz<0.9,>=0.8 in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (0.8.2)
Requirement not upgraded as not directly required: termcolor in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (1.1.0)
Requirement not upgraded as not directly required: msgpack-python in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (0.5.6)
Requirement not upgraded as not directly required: msgpack-numpy==0.4.1 in /usr/local/lib/python3.6/site-packages (from thinc<6.11.0,>=6.10.1->spacy) (0.4.1)
Requirement not upgraded as not directly required: setuptools in /usr/local/lib/python3.6/site-packages (from protobuf>=3.1.0->tensorflow==1.0.0) (39.0.1)
Requirement not upgraded as not directly required: oauthlib>=0.6.2 in /usr/local/lib/python3.6/site-packages (from requests-oauthlib>=0.7.0->tweepy) (2.0.7)
Requirement not upgraded as not directly required: toolz>=0.8.0 in /usr/local/lib/python3.6/site-packages (from cytoolz<0.9,>=0.8->thinc<6.11.0,>=6.10.1->spacy) (0.9.0)`
Have tried deleting all related Dragonfire files and reinstalling countless times to no avail.

**Comment by mertyildiran** [comment]
@Multiboxer I'm not sure what happened on your system. Have you ever tried to install Dragonfire using [install.sh](https://github.com/DragonComputer/Dragonfire/blob/master/install.sh) script? Just run: `sudo ./install.sh`

Not sure why it's installed in Python 2.7, I thought [this definition](https://github.com/DragonComputer/Dragonfire/blob/master/debian/rules#L4) was enough to make it Python 3.x only. Can you try to change [this line](https://github.com/DragonComputer/Dragonfire/blob/master/setup.py#L94) with something Python 3.x ? And if you find the solution, please send a pull request :blush:

**Comment by Multiboxer** [comment]
Changing [that line](https://github.com/DragonComputer/Dragonfire/blob/master/setup.py#L94) to 3.6 and then running `sudo ./install.sh` fixed the missing files issue. Running `dragonfire` then output:
```
Traceback (most recent call last):
  File "/usr/local/bin/dragonfire", line 11, in <module>
    load_entry_point('dragonfire==0.9.9', 'console_scripts', 'dragonfire')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2691, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2322, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2328, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.6/site-packages/dragonfire/__init__.py", line 45, in <module>
    from dragonfire.conversational import DeepConversation
  File "/usr/local/lib/python3.6/site-packages/dragonfire/conversational/__init__.py", line 14, in <module>
    from dragonfire.conversational.textdata import TextData
  File "/usr/local/lib/python3.6/site-packages/dragonfire/conversational/textdata.py", line 15, in <module>
    from dragonfire.conversational.corpus.cornelldata import CornellData
ModuleNotFoundError: No module named 'dragonfire.conversational.corpus'
```
Not sure what caused it to misplace those files and not display any errors, but manually copying `Dragonfire/dragonfire/conversational/corpus/` from the repo to `/usr/local/lib/python3.6/site-packages/dragonfire/conversational/` fixed that and **Dragonfire is now functional.** 👍 Thanks!

**Comment by mertyildiran** [comment]
@Multiboxer could please you send your fix as a pull request? :blush:

**Comment by mertyildiran** [comment]
Fixed with 146f1cd30c1104f109b8756a03def4bd4a659e8c and 6cb545092bdf60f9703cfbdd3f9e97962016be56
